Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -90,6 +91,7 @@ def settingsPanelEntry(panel, parent):
         importFormatsComboBox.addItem(_("default (apply regularization transform)"), "default")
         importFormatsComboBox.addItem(_("none"), "none")
         importFormatsComboBox.addItem(_("apply regularization transform"), "transform")
+        importFormatsComboBox.addItem(_("apply harden regularization transform"), "hardenTransform")
         # In the future additional option, such as "resample" (harden the applied transform) may be added.
```

Comment:
This comment will be out of date so it should be removed with this PR.

Reply:
thanks, I have actually missed it (: